src/Microsoft.ML.Data/Evaluators/AucAggregator.cs

```diff
@@ -112,6 +112,8 @@ public override Double ComputeWeightedAuc(out Double unweighted)
                 }
 
                 Contracts.Check(PosSample != null && NegSample != null, "Must call Finish() before computing AUC");
+                Contracts.CheckNonEmpty(PosSample.ToList(), nameof(PosSample), "AUC is not definied when there is no positive class in the data");
```

Comment:
Converting this to a full list just to check "not empty" feels too heavy. What about using `.Any()` instead?

Reply:
True, will change it

Reply:
done